feature/bgp/static_route_bgp_redistribution/otg_tests/static_route_bgp_redistribution_test/static_route_bgp_redistribution_test.go

```diff
@@ -221,7 +224,7 @@ func configureDUTBGP(t *testing.T, dut *ondatra.DUTDevice) {
 	t.Helper()
 
 	dutOcRoot := &oc.Root{}
-	bgpPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgpPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "default")
```

Comment:
imo this should be a map[string]string variable lookup to enable per-vendor naming schema.

or a deviation-based lookup, similar to how the "static" protocol is currently defined (`static_protocol_name`).

cc @dplore

Reply:
This reference to default network instance should be implemented using deviations.DefaultNetworkInstance()

Reply:
Hi @dplore , just to be sure, are you suggesting the code to change like this :
```bgpPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, deviations.DefaultNetworkInstance(dut))```

Reply:
@dplore 
>This reference to default network instance should be implemented using deviations.DefaultNetworkInstance()

I think this change is about the BGP protocol instance naming (seems that some implementations use `BGP`, while other require the name `default`)

Reply:
Oh, I see, the current code is already using deviations.DefaultNetworkInstance(dut) for the network instance name.   This change is about the bgp instance name.  (thanks for pointing that out  @LimeHat)

The [OC default for this leaf](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-config-name) is `DEFAULT`.   Since this test does not require using a non-default name, the device is expected to use `DEFAULT`.  

The need to use a name which is not `DEFAULT` should be via a deviation.

Reply:
Hi, still waiting to see a deviation specifically for cisco added to use this non-standard  bgp protocol instance name of `default`